.github/workflows/verify_library_generation.yaml

```diff
@@ -3,14 +3,38 @@ on:
     branches:
     - main
   pull_request:
-    paths:
-    - library_generation/**
 
   workflow_dispatch:
 name: verify_library_generation
 jobs:
+  should-run-library-generation-tests:
```

Comment:
What is the difference between this approach and adding `paths` to `pull_request` above?

Reply:
The workflow can only run if the files changed are within `paths`. Therefore, you can't set this workflow as required because it will get stuck and prevent merging for pull requests that don't change files within `paths`.

Reply:
I see, thanks for the explanation, it's very clear to me now! Do you mind adding this info somewhere in the PR description?